src/main/java/simpli/parser/Parser.java

```diff
@@ -0,0 +1,88 @@
+package simpli.parser;
+
+import simpli.actions.Action;
+import simpli.exceptions.ActionException;
+
+import java.util.Arrays;
+
+/**
+ * Parses various strings into an array of tokens.
+ */
+public final class Parser {
+    /**
+     * The maximum number of tokens allowed.
+     *
+     * Token index information:
+     * 0 - command
+     * 1 - isDone
+     * 2 - taskName or taskNum (Mark, Unmark, Delete or Todo, Deadline, Event)
+     * 3 - dueDate or fromDate (Deadline or Event task)
+     * 4 - toDate (Event task)
+     * */
+    private static final int MAX_TOKENS = 5;
+
+    /**
+     * Parses and breaks down the content String into tokens.
+     *
+     * @param content A command String.
+     * @return An Array of tokens.
+     * @throws ActionException if there are unknown tokens or invalid commands.
+     */
+    public String[] parseCommand(String content) throws ActionException {
```

Comment:
I like the way this function handles the parsing of the different user inputs. Great job!